internal/rpchelp/helpdescs_en_US.go

```diff
@@ -474,6 +474,13 @@ var helpDescsEnUS = map[string]string{
 	"importprivkey-rescan":    "Rescan the blockchain (since the genesis block, or scanfrom block) for outputs controlled by the imported key",
 	"importprivkey-scanfrom":  "Block number for where to start rescan from",
 
+	// ImportPubKeyCmd help.
+	"importpubkey--synopsis": "Imports a compressed SEC -encoded public key to the 'imported' account.",
+	"importpubkey-pubkey":   "The 33B compressed SEC public key, hex encoded string",
+	"importpubkey-label":     "Unused (must be unset or 'imported')",
```

Comment:
these don't look alligned to me, formatting issue ?

<img width="1022" alt="image" src="https://user-images.githubusercontent.com/7031569/175814289-c38487dc-b691-4451-9bd9-12a538126339.png">